.github/workflows/create-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Create Plugin Release
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'trunk'
+
+jobs:
+  deploy:
+    if: github.event.pull_request.merged == true && startsWith( github.head_ref, 'release/' )
+    runs-on: ubuntu-latest
+    name: WPJM Release
+    steps:
+      - name: Comment on PR
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: gh pr comment ${{ github.event.number }} --body "🚀 Starting release ..."
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          tools: composer
+          coverage: none
+      - name: Install JS dependencies
+        run: npm ci
+      - name: Install PHP dependencies
+        run: composer install --no-ansi --no-interaction --prefer-dist --no-progress
+      - name: Setup Git
+        run: |
+          git config user.name "WPJM Bot"
+          git config user.email "<robot@wpjobmanager.com>"
+      - name: Create Release
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: |
+          node scripts/create-release.mjs wp-job-manager ${{ github.event.number }}
+          unzip -q wp-job-manager.zip
+      - name: Deploy to WordPress.org
+        uses: 10up/action-wordpress-plugin-deploy@stable
```

Comment:
I think it would be cool to use a fork inside the `Automattic` org for this.

Just to avoid possible security issues.

Reply:
10up is a pretty reputable company and an Automattic partner, so I think it should be safe enough / not worth maintaining a fork.

Reply:
Is @stable the latest version? Would it be possible to at least use a specific version?

Reply:
Pinned this down to a commit hash in [e1cf715](https://github.com/Automattic/WP-Job-Manager/pull/2574/commits/e1cf71558f94f5372f9adcbe456cbcf151a07301)